src/simulation.h

```diff
@@ -7,8 +7,13 @@
 
 #define CHUNK_SIZE 1024
 
+typedef struct history_element {
```

Comment:
I would name this `typedef struct historyElement` for consistency with the rest of the code